requirements-test.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
 #    pip-compile --generate-hashes --output-file=requirements-test.txt requirements-test.in
```

Comment:
`requirements-test-py38.txt` and `requirements-test-py38.in`

Reply:
`requirements-test-py38.txt` is now a link to `requirements-test.txt` thus `requirements-test-py38.in` is no longer needed. In fact it's just another link to `requirements-test.in`

Reply:
If you prefer I can remove the `requirements-test-py38.in` since it has complete no use

Reply:
To better explain what's going on:

The idea is to replace `requirements-py38[-test]*` with `requirements[-test]*` since we're dropping Python 3.6
This could be simply achieved by:
```{bash}
mv requirements-py38.txt requirements.txt
mv requirements-py38.test.txt requirements-test.txt
mv requirements-py38.test.in requirements.test.in
```

**However**, there are 2 problems:
1. if we do that we'll break `iib-playbook` because it's now expecting `requirements-py38.txt` instead of `requirements.txt`, thus I've created symlinks pointing these `-py38` files to the regular ones.
2. the content inside `requirements-py38*` refers to its names and should be changed to the regular files (without suffix `-py38`

That's why I've recompiled the files and created a symlink to them.

But thinking a bit more I figured out your point, it doesn't make sense to keep the files `requirements-py38-test*` because they're just used by DEV env, so I can safely remove them and it will break nothing.

**What I'll do:** I'll send a patched version without the symlinks for `*-py38-test*` since we're not using them and let the link for `requirements-py38.txt -> requirements.txt` exist until we put it back `requirements.txt` in `iib-playbook`